contrib/completion/bash/docker

```diff
@@ -4226,6 +4227,20 @@ _docker_manifest_push() {
 	esac
 }
 
+_docker_manifest_rm() {
+  case "$cur" in
+    -*)
+      COMPREPLY=( $( compgen -S "--help" -- "$cur" ) )
+      ;;
+    *)
```

Comment:
If I understand this feature correctly, the command accepts multiple manifests.
Completion currently is limited to one manifest.
Please simplify this branch to just `__docker_complete_images --force-tag --id`